wordpress-mobile/WordPress-iOS
# Activity Log Remote Feed initial implementation and tests

**First step in** #7832
Remote implementation of the activity log feed endpoint.

**To test:**
Execute the tests, see that all is green.
For a site that supports the Activity Log you can add the following code to invoke the remote fetch:
```
let activityServiceRemote = ActivityServiceRemote(wordPressComRestApi: blog.wordPressComRestApi())
  activityServiceRemote?.getActivityForSite((blog.dotComID?.intValue)!, count: 20, success: { (activities, hasMore) in
    print(activities.count, hasMore)
  }, failure: { (error) in
    print("Error \(error)")
})
```
Needs review: @koke
